src/index.js

```diff
@@ -107,6 +108,18 @@ function start () {
     .then(onResourcesLoaded)
 }
 
+function initializeApiListener (store: Object) {
+  OvirtApi.addHttpListener((requestId, eventType) => {
+    if (eventType === 'START') {
+      store.dispatch(addActiveRequest(requestId))
+      return
+    }
+    if (eventType === 'STOP') {
+      store.dispatch(delayedRemoveActiveRequest(requestId))
```

Comment:
So the purpose of this is to leave the "Loading..." indicator on screen for an extra half second?  Is that to prevent flashing if a bunch of XHR requests are done one after another?

Reply:
yes

Reply:
@sjd78 as mentioned in the commit message your worries came true